src/components/Label/Label.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../../styles/mixins';
```

Comment:
You have to import this file to styles.scss where all of the styles are connected

Reply:
File imported to styles.scss